### PR TITLE
Use new metadata locations for fdroid

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Once https://gitlab.com/fdroid/fdroidserver/merge_requests/370 is merged, we can use

/src/generic/fastlane/metadata/android/$locale
/src/versionDev/fastlane/metadata/android/$locale

to generate metadata infos in fdroid store.

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
You just deleted the metadata files, does that mean they will "come back" via the translation bot?

**Comment by tobiasKaminsky** [comment]
Yes, but in this branch they will be in /src/generic/fastlane/... and /src/versionDev/fastlane/...

**Comment by mario** [comment]
So this needs lots of rebase magic?

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky can you take a look at this? Rebase shouldn't be too complicated as long as you know how to resolve the conflicts in the build files. :/

File: build.gradle [hunk 1]
```diff
@@ -193,6 +193,8 @@ dependencies {
     implementation 'com.android.support:multidex:1.0.2'
     implementation 'com.github.nextcloud:android-library:1.0.33'
     versionDevImplementation 'com.github.nextcloud:android-library:1.0.33'
+    versionDevImplementation 'com.github.nextcloud:android-library:master-SNAPSHOT'
+    versionDevImplementation 'com.github.nextcloud:android-library:master-SNAPSHOT'
```
Comment (AndyScherzinger) [inline build.gradle]: Duplicate entry ```versionDevImplementation 'com.github.nextcloud:android-library:master-SNAPSHOT'```

**Comment by tobiasKaminsky** [comment]
/src/fastlane folder is removed

Instead we use:
/src/generic/fastlane/metadata/android/$locale
/src/versionDev/fastlane/metadata/android/$locale

As fallback we need to have
/src/generic/fastlane/metadata/android/en-US/title.txt
/src/versionDev/fastlane/metadata/android/en-US/title.txt

From now on everything will be translated in transifex and will be back again on next transifex bot run.

**Comment by AndyScherzinger** [comment]
Fine by me, only thing I saw in the f-droid client is a linebreak issue: https://github.com/nextcloud/android/issues/1081#issuecomment-350997570

**Comment by tobiasKaminsky** [comment]
I will try to change this.
Please do NOT merge

**Comment by tobiasKaminsky** [comment]
Linebreaks should be fixed now

**Comment by AndyScherzinger** [comment]
lint score is up by +1 warnings, maybe another rebase might help (or there is actually a new warning).

**Comment by AndyScherzinger** [comment]
👍 fine by me

**Comment by mario** [comment]
Rebase.

**Comment by tobiasKaminsky** [comment]
Done :-)

File: src/main/res/values/strings.xml [hunk 2]
```diff
@@ -682,7 +682,7 @@
     <string name="resized_image_not_possible">No resized image available. Download full image?</string>
     <string name="resized_images_download_full_image">Download full image?</string>
 
-    <string name="store_short_desc">The Nextcloud Android app gives you access to all your files in your Nextcloud</string>
+    <string name="store_short_desc">The Nextcloud Android app allows you to access all your files on your Nextcloud</string>
```
Comment (mario) [inline src/main/res/values/strings.xml]: Suggestion: The Nextcloud Android app allows access to all files on your Nextcloud instance.

Seems simpler at least to me.

**Comment by tobiasKaminsky** [comment]
@mario thanks for the wording feedback, changed.
Can you approve it?
After that I can rebase & merge it.

**Comment by mario** [comment]
👍

**Comment by mario** [comment]
Done!

**Pull request merged**